Assets/MediaSoup/HandlerInterface.cs

```diff
@@ -440,17 +445,17 @@ public virtual async Task<HandlerSendResult> Send(HandlerSendOptions options)
         var stats = await GetPcStats(pc);
         string statsString = JsonConvert.SerializeObject(stats);
 
-        Debug.Log("Manually adding ice candidates: ");
+        //Debug.Log("Manually adding ice candidates: ");
 
-        foreach (IceCandidate candidate in iceCandidates)
-        {
-            RTCIceCandidateInit init = ConvertMediasoupToUnityIceCandidate(candidate, iceParameters);
-            RTCIceCandidate iceCandidate = new RTCIceCandidate(init);
-            bool isIceCandidateAdded = pc.AddIceCandidate(iceCandidate);
-            Debug.Log($"IceCandidate: {iceCandidate}, Candidate: {iceCandidate.Candidate}, Type: {iceCandidate.Type}, Address: {iceCandidate.Address}, port: {iceCandidate.Port}, username: {iceCandidate.UserNameFragment}, isIceCandidateAddded: {isIceCandidateAdded}, ");
-        }
+        //foreach (IceCandidate candidate in iceCandidates)
+        //{
+        //    RTCIceCandidateInit init = ConvertMediasoupToUnityIceCandidate(candidate, iceParameters);
+        //    RTCIceCandidate iceCandidate = new RTCIceCandidate(init);
+        //    bool isIceCandidateAdded = pc.AddIceCandidate(iceCandidate);
+        //    Debug.Log($"IceCandidate: {iceCandidate}, Candidate: {iceCandidate.Candidate}, Type: {iceCandidate.Type}, Address: {iceCandidate.Address}, port: {iceCandidate.Port}, username: {iceCandidate.UserNameFragment}, isIceCandidateAddded: {isIceCandidateAdded}, ");
+        //}

```

Comment:
Removed this to avoid getting the ice connection state getting stuck in checking